app/tests/callback_test.cc

```diff
@@ -290,13 +290,13 @@ TEST_F(CallbackTest, CallCallbackValue2String1) {
   EXPECT_THAT(value_and_string_.second, Eq("meaning"));
 }
 
-// Call a callback passing the UniquePtr
+// Call a callback passing the std::unique_ptr
```

Comment:
I see this tests using a unique_ptr in the Callback logic, but we were concerned about that wrt how query_desktop was using it.  Any concerns?  Should we add a test for the whole shared wrapping unique thing?

Reply:
Good idea, I'll add it.

Reply:
Test added, PTAL.